messagingpush/src/main/java/io/customer/messagingpush/ModuleMessagingPushFCM.kt

```diff
@@ -9,6 +11,8 @@ import io.customer.sdk.CustomerIOInstance
 import io.customer.sdk.di.CustomerIOComponent
 import io.customer.sdk.module.CustomerIOModule
 
+@VisibleForTesting
+@InternalCustomerIOApi
```

Comment:
Both of these annotations should go on `constructor` and not on class I think 🤓

Reply:
🤦